CHANGES.md

```diff
@@ -1,5 +1,33 @@
 # Changes
 
+## stable-2.11.3
+
+This release pulls in several control plane and proxy fixes from the main
+development branch. The linkerd-multicluster extension has several fixes
+regarding incorrect label matching and resource cleanup. Additionally, a long
+standing panic has been fixed in the proxy.
+
+* Fixed an error in `linkerd multicluster allow` which resulted in broken YAML
+  output
+* Fixed a panic in the proxy in cases where the destination service sends a very
+  large number of discovery updates
```

Comment:
This is semantics, but I don't think the panic was really limited to cases where the destination service sends a large number of discovery updates, though that is one scenario that could trigger the panic.

This is how I phrased it in the tagged release:

```suggestion
* Fixed a potential panic in the proxy's outbound load balancer that
  could be triggered when the balancer processes many service
  discovery updates in a short period of time.
```